package.json

```diff
@@ -20,7 +20,7 @@
     "@visx/grid": "1.16.0",
     "@visx/group": "1.7.0",
     "@visx/scale": "1.7.0",
-    "@visx/shape": "1.8.0",
+    "@visx/shape": "2.12.2",
```

Comment:
This is updated to support `<Pie />`'s [`fill` attribute being a function](https://airbnb.io/visx/docs/shape#Pie_fill). Previously, it just accept strings.